tests/test_examples.py

```diff
@@ -0,0 +1,105 @@
+# License: BSD 3 clause
+"""
+Run the examples, just to make sure they are all still working
+
+:author: Jeremy Biggs (jbiggs@@ets.org)
+"""
+
+from glob import glob
+from os import chdir, getcwd, listdir, makedirs
+from os.path import abspath, basename, dirname, join
+from shutil import copyfile, rmtree
+
+from examples import make_titanic_example_data as mtd
+from examples import make_boston_example_data as mbd
+from examples import make_iris_example_data as mid
+
+from skll.experiments import run_configuration
+
+
+_my_cwd = getcwd()
+_my_dir = abspath(dirname(__file__))
+_examples_dir = join(_my_dir, '..', 'examples')
+
+_old_titanic_dir = join(_examples_dir, 'titanic')
+_old_boston_dir = join(_examples_dir, 'boston')
+_old_iris_dir = join(_examples_dir, 'iris')
+
+_new_titanic_dir = join(_my_dir, 'other', 'titanic')
+_new_boston_dir = join(_my_dir, 'other', 'boston')
+_new_iris_dir = join(_my_dir, 'other', 'iris')
+
+
+def run_function_in_custom_dir(func, old_dir, new_dir):
```

Comment:
This seems fine for now. Although it might be nice to fix these scripts so that they don't require being run from a particular directory so that this wouldn't be necessary.

Reply:
Could we just use `subprocess.run()` here to run the script which allows specifying the working directly explicitly?